string messageToPrint
float32 someFloatValue
---
string messageBackToClient
